CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/SyntaxNodesFile.swift

```diff
@@ -304,3 +309,27 @@ private func generateSyntaxChildChoices(for child: Child) throws -> EnumDeclSynt
     }
   }
 }
+
+private func generateInitializerDocComment(for node: LayoutNode) -> SwiftSyntax.Trivia {
+  func generateParamDocComment(for child: Child) -> String? {
+    guard let documentation = child.documentation,
+      let firstLine = documentation.split(whereSeparator: \.isNewline).first
+    else {
+      return nil
+    }
+
+    return "   - \(child.varName): \(firstLine)"
+  }
+
+  let paramsDoc = docCommentTrivia(
+    from: ([
+      " - Parameters:",
+      "   - leadingTrivia: The leading ``Trivia`` for this node.",
```

Comment:
The semantics of `leadingTrivia` and `trailingTrivia` are a little subtle and I’m happy that we’re adding a doc comment for it now. Could you change the comment to

```
Trivia to be prepended to the leading trivia of the node’s first token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
```

Similarly, the documentation for `trailingTrivia` should be 

```
Trivia to be appended to the trailing trivia of the node’s last token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
```